### ISSUE TITLE
Text under image is not displayed

### ISSUE THREAD

**Issue by mentalo-** [SilenceIM/Silence#680]
### Bug description
I use the last version of Silence, on Android 6.0.
The other person use the native SMS/MMS app on Android 6.0.

### How to reproduce
Using Silence : 
- in a conversation, tap on the "attachment" button
- search an image et join it
- below the image, write a text
- send the message

**Actual result:** the image is received, but not the text
**Expected result:** the text is displayed below the image

### Screenshots
Silence doen't allow me to take a screenshot 

### Device info
- Device: Redmi 3S
- Android version: 6.0
- Silence version: 0.15.13

### Link to debug log
sorry I don't find any log

**Comment by licaon-kter** [comment]
Sending the same to yourself works in Silence?

**Comment by mentalo-** [comment]
Yes I see the text

**Comment by williamdes** [comment]
I think I had this issue.
Set the text before joining the image works.
Can you confirm @mentalo-

**Comment by mentalo-** [comment]
No, the recipient of my message didn't receive anything (test setting the text before or after joining the image)
